### DISCUSSION TITLE
generating a Test Log report based on the log-identifier

### DISCUSSION THREAD

**Discussion by fadihawli** [testpointcorp/Vansah-API-Binding-Java#9] [General]
We need to be able to generate a report at any time using the "log_identifier" value.

Example the current report will be sent as a link with the Result/test properties in the subject field.

![image](https://user-images.githubusercontent.com/8530749/102422898-da15de80-405b-11eb-8aa2-f16f8ff0c051.png)

@tp-Fernando  how do you currently get the report of a test case. lets say it fails and you need to be alerted ?

**Comment by tp-Fernando** [comment]
@fadihawli  We usually generate the report at the end of the execution by calling https://vansahapp.net/api/report/logs/. I can filter this report by failed ones and understand what went wrong. No alerts though.
